styles.css

```diff
@@ -915,7 +923,7 @@ nav:hover {
     background: #555;
   }
 
-  .mobile-container .Project_work .workform4 button {
+  .mobile-container .Project_work .workform4 .button4 {
```

Comment:
- [ ] Kindly fix the popup for the mobile version the (x) button is not showing and is not well style.
![Capturepopu](https://user-images.githubusercontent.com/81444338/175490294-5a765bd3-28ad-4655-930f-8ed7bc676d86.PNG)